### PR TITLE
utils_selinux: add some useful options for setting selinux context

### PR DESCRIPTION
**Pull request by PandaWei**
1) add option '-r' used to set MLS/MCS Security Range
   for command semanage.
2) add option '-F' used to force a replacement of the entire context
   for command restorecon.

### CODE REVIEW

File: virttest/utils_selinux.py [hunk 1]
```diff
@@ -352,28 +358,31 @@ def del_defcon(context_type, pathregex):
...
 
     :param pathname: Absolute path to file, directory, or symlink
     :param dirdesc: True to descend into sub-directories
+    :param repforce: True to force a replacement of the entire context
     :return: True if all components match default contexts
     :note: By default DOES NOT follow symlinks
     """
     # Default context path regexes only work on canonical paths
-    changes = _run_restorecon(pathname, dirdesc)
+    changes = _run_restorecon(pathname, dirdesc, readonly=readonly, force=forcedesc)
```
Comment (yumingfei) [inline virttest/utils_selinux.py]: It may exceed 80 characters.

Comment (PandaWei) [inline virttest/utils_selinux.py]: Hi, 
devided it into two lines.

File: virttest/utils_selinux.py [hunk 2]
```diff
@@ -352,28 +358,32 @@ def del_defcon(context_type, pathregex):
...
 
 
-def verify_defcon(pathname, dirdesc=False):
+def verify_defcon(pathname, dirdesc=False, readonly=True, forcedesc=False):
     """
     Verify contexts of pathspec (and/or below, if dirdesc) match default
 
     :param pathname: Absolute path to file, directory, or symlink
     :param dirdesc: True to descend into sub-directories
+    :param repforce: True to force a replacement of the entire context
```
Comment (will-Do) [inline virttest/utils_selinux.py]: Patch is good, but I hope you can complete the doc for all these parameters.

Comment (PandaWei) [inline virttest/utils_selinux.py]: Hi,
made some complements as your comment

Comment (will-Do) [inline virttest/utils_selinux.py]: Well, I mean all parameters you added

Comment (PandaWei) [inline virttest/utils_selinux.py]: OK, Thanks for reminder
I added two options,
One is  '-r ' for semanage-fcontext
the other is '-F' for restorecon

Now,  all new parameters have been  annotated

Comment (PandaWei) [inline virttest/utils_selinux.py]: OK, Thanks for reminder
I added two options,
One is  '-r ' for semanage-fcontext
the other is '-F' for restorecon

Now,  all new parameters have been annotated

**Comment by will-Do** [comment]
Thanks, ACK this patch

**Comment by yumingfei** [comment]
Acked, thanks.

**Pull request merged**